bin.js

```diff
@@ -146,6 +147,10 @@ function start (entry, argv, done) {
   var address = argv.address
   var port = argv.port
   var sse = Sse(assets)
+  var certOpts = {
+    installCertutil: true
+  }
+  var certName = entry.replace(new RegExp(path.sep, 'g'), '-').slice(-64)
```

Comment:
The certname should be unique for each project (maybe per entry point?) -- but there's a 64 character limit on it due to ASN1:

```
↳ openssl req -config /Users/todd/.config/devcert/openssl.conf -subj "/CN=-Users-todd-src-showrunner-node_modules-@scriptollc-code-mirror-ot-standalone.js" -key /Users/todd/.config/devcert/-Users-todd-src-showrunner-node_modules-@scriptollc-code-mirror-ot-standalone.js.key -out /Users/todd/.config/devcert/-Users-todd-src-showrunner-node_modules-@scriptollc-code-mirror-ot-standalone.js.csr -new
problems making Certificate Request
4550:error:0D07A097:asn1 encoding routines:ASN1_mbstring_ncopy:string too long:/BuildRoot/Library/Caches/com.apple.xbs/Sources/OpenSSL098/OpenSSL098-64.50.6/src/crypto/asn1/a_mbstr.c:154:maxsize=64
```